main.py

```diff
@@ -337,6 +342,7 @@ def disable_lldp(self):
             else:
                 error_list.append(id_)
         if not error_list:
+            self.notify_lldp_change()
```

Comment:
It would be better if the event was not an empty one. It could include some information about which interface was modified, and if it was enabled or disabled.

Reply:
I added information about the modified interface at the event. These modifications also changed the PR https://github.com/kytos/topology/pull/111